src/SpiceSharpParser/Common/Mathematics/Probability/IRandomNumberProvider.cs

```diff
@@ -6,6 +6,8 @@ public interface IRandomNumberProvider : IRandomIntegerProvider, IRandomDoublePr
 
     public interface IRandomDoubleProvider
     {
+        double NextSignedDouble();

```

Comment:
How does this method differ from `NextDouble` for a _general_ random number provider?

Reply:
@svenboulanger  I've found in PSpice and other documation that  .DISTRIBUTION dot statement specifies random number that in in range (-1,1). I would like to have method to easy generate number in that range.

Reply:
Then is a number generator between (0,1) still necessary? And does it have to be produced in the same `IRandomDoubleProvider` or is it more logical to have *two* random double providers: one giving you (-1, 1) and one producing (0, 1)?

Reply:
@svenboulanger  I think it's not necessary at this moment to split it into two providers. 
The changes that I want to commit it's the first step in providing good randomness support in SpiceSharpParser.

Reply:
@svenboulanger  There will be refactor for sure in the near future.